src/defaultsettings.cpp

```diff
@@ -410,6 +410,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("TMPFolder","/sdcard/" PROJECT_NAME_C "/tmp/");
 	settings->setDefault("touchscreen_threshold","20");
 	settings->setDefault("fixed_virtual_joystick", "false");
+	settings->setDefault("virtual_joystick_controls_use", "false");
```

Comment:
Maybe 'virtual_joystick_triggers_aux'? 'controls' is a little awkward sounding and has a double meaning.

Reply:
Okay.